pontoon/insights/views.py

```diff
@@ -16,17 +17,25 @@ def insights(request):
     if not settings.ENABLE_INSIGHTS:
         raise ImproperlyConfigured("ENABLE_INSIGHTS variable not set in settings.")
 
+    team_pretranslation_quality = cache.get_or_set(
+        f"/{__name__}/team_pretranslation_quality",
+        get_global_pretranslation_quality("locale", "code"),
+        settings.VIEW_CACHE_TIMEOUT,
+    )
+
+    project_pretranslation_quality = cache.get_or_set(
+        f"/{__name__}/project_pretranslation_quality",
+        get_global_pretranslation_quality("entity__resource__project", "slug"),
+        settings.VIEW_CACHE_TIMEOUT,
+    )
```

Comment:
Why write these in the cache? This appears to be the only place where the values are used, and with `get_or_set()` the `get_global_..._quality()` function is always called.